pkg/collector/stats/types/types.go

```diff
@@ -14,10 +14,10 @@ type UInt64Stat struct {
 }
 
 func NewUInt64Stat(aggr, delta uint64) *UInt64Stat {
-	stat := &UInt64Stat{}
+	stat := UInt64Stat{}
```

Comment:
fix `Error: pkg/collector/stats/types/types.go:17:10: &UInt64Stat{} escapes to heap`
ref https://github.com/sustainable-computing-io/kepler/actions/runs/12177539929/job/33966954289
and
https://github.com/sustainable-computing-io/kepler/actions/runs/12197388859/job/34026929446?pr=1886